pkg/apis/federatedscheduling/v1alpha1/zz_generated.conversion.go

```diff
@@ -1,7 +1,7 @@
 // +build !ignore_autogenerated
 
 /*
-Copyright 2018 The Federation v2 Authors.
+Copyright 2018 The Kubernetes Authors.
```

Comment:
@shashidharatd It appears this code needs to be regenerated to reflect this change to the boilerplate made when the repo was moved.  I've placed a `do-not-merge` label accordingly.

@pmorie Were you going to add a boilerplate check?

Reply:
@marun The new boilerplate specifies `The Kubernetes Authors` and is correct now that we are part of the k8s org. I asked @pmorie in https://github.com/kubernetes-sigs/federation-v2/pull/77#issuecomment-391050970 to verify because I had the same concern. That PR also added a boilerplate checker that runs as part of CI.